ext4: remove redundant code in ext4_fill_super()

From: Li Zefan <lizf@cn.fujitsu.com>

The previous sb_min_blocksize() has already set the block size.

Signed-off-by: Li Zefan <lizf@cn.fujitsu.com>
Signed-off-by: Mingming Cao <cmm@us.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
---
 fs/ext4/super.c |    5 -----
 1 file changed, 5 deletions(-)

Index: linux-2.6.26-rc9/fs/ext4/super.c
===================================================================
--- linux-2.6.26-rc9.orig/fs/ext4/super.c	2008-07-11 16:05:01.000000000 -0700
+++ linux-2.6.26-rc9/fs/ext4/super.c	2008-07-11 16:05:02.000000000 -0700
@@ -1899,11 +1899,6 @@ static int ext4_fill_super (struct super
 		goto out_fail;
 	}
 
-	if (!sb_set_blocksize(sb, blocksize)) {
-		printk(KERN_ERR "EXT4-fs: bad blocksize %d.\n", blocksize);
-		goto out_fail;
-	}
-
 	/*
 	 * The ext4 superblock will not be buffer aligned for other than 1kB
 	 * block sizes.  We need to calculate the offset from buffer start.
